.github/workflows/ci_master.yml

```diff
@@ -1,39 +1,30 @@
-name: Build
-
+name: CI Master
```

Comment:
This file seems to be identical to `ci.yml`, save for the workflow trigger.

Would it make sense to remove this duplication and move the trigger over to `ci.yml`, rather than having a dedicated file for master branch's CI?

Reply:
We can do that. 
But, How will be able to distinguish running the CD process, only on master branch ? 

The `workflow_run:` triggers on workflow complete.

Reply:
There seems to be a way to control which branches' workflows the `workflow_run` trigger should react to, e.g. by adding the `branches: [master]` key/value to the trigger: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#limiting-your-workflow-to-run-based-on-branches

Reply:
@mvitale1989  Pushed https://github.com/mattermost/docs/pull/6604/commits/61ca6e84af197fc5015ab2deb02904bc3a365f2d to address this.